.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+# Pull Request
+
+## Summary
+
+Provide a concise and clear summary of your pull request.
+
+## Motivation
+
+Explain the motivation behind this pull request. What problem does it solve or feature does it introduce?
+
+## Changes Made
+
+Describe the changes made in this pull request. Be specific about code changes, new features, or bug fixes.
+
+## Related Issues/PRs
+
+Link to any related issues or pull requests, if applicable.
+
+## Screenshots (Before/After if applicable)
+
+Include screenshots or visual aids to demonstrate the changes visually. If your PR modifies the UI or behavior, include before and after descriptions or screenshots.
+
+### Desktop
+
+### Mobile
+
+### iPad
```

Comment:
Suggestion: No need for these Desktop / mobile / iPad sections because this tool is headless (no UI provided). :)

Reply:
Just went with the flow of standard PR templates in frontend projects 😅
Removed 👍🏼